docs/data/charts/styling/styling.md

```diff
@@ -147,12 +147,15 @@ Those will fix the chart's size to the given value (in px).
 
 ### Placement
 
-At the core of chart layout is the drawing area which corresponds to the space available to represent data.
+There are two concepts to consider when defining the placement of a chart:
 
-This space can be defined with the `margin` prop and its properties `top`, `bottom`, `left`, and `right`.
-Those values define the space between the SVG border and the drawing area.
+- **`margin`**: The space between the SVG border and the axis or drawing area.
+- **`axis size`**: The space taken by the [axis](/x/react-charts/axis/#position). Each axis has their own size.
 
-You might want to modify those values to leave more space for your axis ticks or reduce them to provide more space for the data.
+To define the axis size, you can use the `xAxis` and `yAxis` configuration. The size is only take into account if the axis has a position different from `none`.
+
+- **`x-axis`**: Uses the `height` prop to calculate the space taken by the axis.
```

Comment:
This is a bit of a nit, but do we calculate the space taken by the axis? I thought we just used the provided value directly.

Reply:
We don't calculate it, but if it's not clear for you i=t might not be obvious for users either

Reply:
The word "calculate" here makes me feel like the value I provided might be changed in some conditions. If the value is always used as is, I think a different wording might be clearer, e.g., "Uses the `height` prop to define the space taken by the axis."